features/cxf/src/main/feature/feature.xml

```diff
@@ -541,10 +471,9 @@
         <feature version="${cxf.version}">cxf-jaxrs</feature>
         <feature version="${cxf.version}">cxf-databinding-jaxb</feature>
         <feature version="${cxf.version}">cxf-databinding-aegis</feature>
-        <feature version="${cxf.version}">cxf-databinding-xmlbeans</feature>
         <feature version="${cxf.version}">cxf-bindings-corba</feature>
         <feature version="${cxf.version}">cxf-bindings-coloc</feature>
-        <feature version="${cxf.version}">cxf-bindings-object</feature>
+        <!--<feature version="${cxf.version}">cxf-http-provider</feature>-->
         <feature version="${cxf.version}">cxf-http-jetty</feature>
```

Comment:
I should have added this line back in. It is not present in this version of CXF, as the `cxf-http-jetty` feature is presumed to start with `cxf-http-provider`.